app/src/main/java/com/kickstarter/ui/activities/SetPasswordActivity.kt

```diff
@@ -88,8 +99,4 @@ class SetPasswordActivity : BaseActivity<SetPasswordViewModel.ViewModel>() {
     private fun setFormDisabled(isDisabled: Boolean) {
         setFormEnabled(!isDisabled)
     }
-
-    override fun back() {
```

Comment:
@leighdouglas I think the disable back action was on purpose, to not allow the user to leave the screen

Reply:
Good catch, there is no back button, but a user can swipe or press the backbutton to exit. I'll bring this up during sprint planning and add this as a QA script.